src/selectors/entities/users.ts

```diff
@@ -344,6 +353,17 @@ export function searchProfiles(state: GlobalState, term: string, skipCurrent = f
     return filteredProfiles;
 }
 
+export function searchProfilesInChannel(state: GlobalState, channelId: $ID<Channel>, term: string, skipCurrent = false): Array<UserProfile> {
+    const doGetProfilesInChannel = makeGetProfilesInChannel();
+    const profiles = filterProfilesMatchingTerm(doGetProfilesInChannel(state, channelId, false), term);
```

Comment:
```suggestion
    const profiles = filterProfilesMatchingTerm(doGetProfilesInChannel(state, channelId), term);
```
I think you can remove the `false` since the function by default is false for that parameter.
![image](https://user-images.githubusercontent.com/17804942/81760418-229d6600-9495-11ea-99c0-ff751ec09e89.png)